geoprocessing/test/integration/protected-areas/steps/invalid-shapefile.ts

```diff
@@ -0,0 +1,27 @@
+import { readFileSync } from 'fs';
+import { Readable } from 'stream';
+
+export const shapePath = __dirname + `/new-shape-name.zip`;
```

Comment:
I am not sure we can easily make this setup work for tests running under Docker - I suggest to first copy the files to `/tmp`.

The issue is that we bind-mount `src` and `test` inside the container, and this ends up mounting them as a user other than the one that owns the `node` process for the NestJS service.

I have seen suggestions to *`mkdir` and `chown`* directories that are then used for bind mounts, but in practice we already do this via the `COPY --chown ...` commands in the `Dockerfile`s, and in any case none of this works at least with the Docker engine version I am using locally nor the one we have in the GH Actions runner.

The `config` directory gets copied over with correct ownership, and so should do `src` and `test`, but once we bind mount volumes, the owner gets overridden with the owner of the source directory of the volume in the host filesystem.

Reply:
```
[Nest] 133   - 05/25/2021, 1:43:59 PM   [ExceptionHandler] Error: new-shape-name could not be extracted: Error: Error: EACCES: permission denied, mkdir '/opt/marxan-geoprocessing/test/integration/protected-areas/steps/new-shape-name' +130ms
``` 
indeed, I didn't catch it 🤔 

I don't feel fine to modify the tests just due to different environment (would be those test reliable?)
If we are going to use shared volume, those will change anyway "a bit", so I struggle to figure out how to make move forwards without going back and forth 🤔 

What do you propose @hotzevzl ?

Reply:
(just to mention: running those specs locally via docker-compose works nice 🤔 )

Reply:
hm, it actually fails not on reading a file but while trying to unpack it

Reply:
![image](https://user-images.githubusercontent.com/3466388/119620325-27655b80-be05-11eb-80f4-616332a93cce.png)
as I mentioned a few times, we should probably unpack it to %TMP folder instead of the very same location

Reply:
I am out of ideas for now. @hotzevzl should we mark those tests as `skipped` and move forwards until we manage to figure out shared volume?

Reply:
@kgajowy (cc @alexeh) yes, I think we should unpack within a temporary folder, letting the unpacker code (or something close to it) take care of this, so that the callers don't need to know about details of the filesystem (at least some of them... we need to draw a line somewhere for MVP).

For now let's `.skip`, but we need to add `tmp` handling to the unpacker or so in order to enable these tests - shared volume won't by itself help with this - we'd still need to unpack the contents of zip files outside of the `source`/`test` subtrees.

BTW in practice these tests *could* work as they are *iff* the owner of the `test` folder *on the host machine* is set to 5000, but we cannot rely on this.